CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
-# varnish Cookbook CHANGELOG
+# Changelog
```

Comment:
Why are we moving away from having Varnish in the header? 🤔

Reply:
Just copying from the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) site but I am ok with either.